src/pages/patientView/clinicalInformation/ClinicalInformationPatientTable.tsx

```diff
@@ -36,6 +37,36 @@ export default class ClinicalInformationPatientTable extends React.Component<ICl
         }
         return ret;
     }
+    
+    handleClick = (name:string) => {
```

Comment:
this should be methods, not functions assigned to properties.    so, handleClick() { ... }
You can then apply decorators like @bind or @action to them

Reply:
just to clarify, I think @alisman means `handleClick(name:string) { ... }`

Reply:
yes